src/Main.h

```diff
@@ -65,15 +65,25 @@ class ATTRIBUTE_HIDDEN CVisualizationGoom
   void OnCompiledAndLinked() override;
   bool OnEnabled() override;
 
+protected:  
+  virtual void NoActiveBufferAvailable() {}
+  virtual void AudioDataQueueTooBig() {}
+  virtual void SkippedAudioData() {}
+  virtual void UpdateGoomBuffer(
```

Comment:
What was thought by add of this virtual functions?
Planned something about a child of this class?